src/modules/Requests.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const Requests = {
+  async searchSelfCare(searchQuery) {
+    const response = await axios.get(`/api/search?q=${searchQuery}`, {
+      headers: { API_KEY: process.env.REACT_APP_API_KEY },
```

Comment:
There's some code repetition concerning headers. Why not set this in the `index.js` (or a config file). like this:

```
axios.defaults.headers.common['API_KEY'] = process.env.REACT_APP_API_KEY
```

Reply:
that is cool! changed